### PR TITLE
Issue 4099: Make zookeeper image configurable for system tests

### LINKED ISSUE #4099: Zookeeper Image used by system tests cannot be changed
**Problem description**
Currently system tests cannot accept zookeeper image version as a parameter and always install the default zookeeper version deployed by zookeeper-operator.
We should be able to run system tests with different versions of zookeeper and for this system tests gradle target should be able to accept zookeeper-image repository and tag values as parameters from command line.

**Problem location**
Need to fix gradle task startK8SystemTests and system test framework code to make zookeeper-image pluggable.

**Suggestions for an improvement**

### PR DESCRIPTION
**Pull request by anishakj**
**Change log description**  
Added zookeeper image as a configurable parameter for system tests

**Purpose of the change**  
Fixes #4099

**What the code does**  
Added zookeeper image param in gradle. 
And will use this to start zookeeper pod. If param is not mentioned will use the latest image

**How to verify it**  
System tests should pass

### CODE REVIEW

**Comment by shrids** [comment]
We should also ensure the zookeeper image version for the docker based system tests is configurable too.

**Comment by anishakj** [comment]
Made changes for zookeeper image version configuration for docker swarm tests

File: test/system/src/main/java/io/pravega/test/system/framework/services/docker/ZookeeperDockerService.java [hunk 1]
```diff
@@ -58,7 +58,7 @@ private ServiceSpec setServiceSpec() {
 
         final TaskSpec taskSpec = TaskSpec
                 .builder()
-                .containerSpec(ContainerSpec.builder().image(ZK_IMAGE)
+                .containerSpec(ContainerSpec.builder().image(IMAGE_PATH + IMAGE_PREFIX + "/zookeeper:" + ZK_VERSION)
```
Comment (shrids) [inline test/system/src/main/java/io/pravega/test/system/framework/services/docker/ZookeeperDockerService.java]: we should make the image name configurable.

Comment (anishakj) [reply test/system/src/main/java/io/pravega/test/system/framework/services/docker/ZookeeperDockerService.java]: Made the changes

**Comment by anishakj** [comment]
Both K8 and Docker swarm system tests are passed with the changes.

File: build.gradle [hunk 2]
```diff
@@ -833,6 +833,8 @@ project('test:system') {
         systemProperty "imagePrefix", System.getProperty("imagePrefix", "pravega")  // the default value of imagePrefix is pravega.
         systemProperty "pravegaImageName", System.getProperty("pravegaImageName", "pravega") // pravega image name
         systemProperty "bookkeeperImageName", System.getProperty("bookkeeperImageName", "bookkeeper") // bookkeeper image name
+        systemProperty "zookeeperImageName", System.getProperty("zookeeperImageName", "zookeeper") // zookkeeper image name
+        systemProperty "zookeeperVersion", System.getProperty("zookeeperVersion", "latest") // zookeeper image version
```
Comment (shrids) [inline build.gradle]: `zookeeperImageVersion` seems more apt here and eliminates the confusion of it referring the ZK version. Could we change it ?

Comment (anishakj) [reply build.gradle]: Changed that

**Pull request merged**